.scalafix.conf

```diff
@@ -34,3 +39,17 @@ Disable.ifSynthetic = [
     message = "not a total function"
   }
 ]
+
+ExplicitResultTypes {
+  unsafeShortenNames = true
```

Comment:
What does it do?

Reply:
apparently it's been removed from current version of Scalafix https://scalacenter.github.io/scalafix/docs/rules/ExplicitResultTypes.html